Sources/BraveNews/FeedCardContent.swift

```diff
@@ -38,3 +38,11 @@ public enum FeedItemAction: Equatable {
   /// Toggle the enabled status for the source of the feed item
   case toggledSource
 }
+
+/// The actions you can perform on a rate card item
+public enum RatingCardAction {
+  // The user choose to navigate Appstore for app rating
+  case rateBrave
+  // The user choose to hide card for app rating
+  case hideCard
+}
```

Comment:
This can probably go in the RatingCardsView file

Reply:
Placed RatingCardView https://github.com/brave/brave-ios/pull/7589/commits/ebbe26fa4af9e4be47f82ac8e078025b0f0f5031